app/assets/stylesheets/pageflow/themes/default/mobile_navigation.scss

```diff
@@ -100,5 +100,13 @@ $mobile-navigation-pictogram-visibility: $standard-page-pictogram-visibility !de
     svg {
       fill: $mobile-navigation-share-link-color;
     }
+
+    &.active {
+      color: $main-color;
+
+      svg {
+        fill: $main-color;
```

Comment:
Shall we also introduce a new variable here to allow controlling this color individually like you did in the other PR?

Reply:
Outline navigation bar only defines a handful of variables exactly for coloring, that's why I added the equivalent variables for sharing there.
In pageflow there's already so many SCSS-variables that I opted for using the main color instead of introducing yet more variables.

Reply:
I think SCSS-variables are a good thing actually. They are the public API of our theme.